.github/renovate.json

```diff
@@ -29,8 +34,25 @@
         "^multi-semantic-release$"
       ],
       "groupName": "semantic-release monorepo"
-    }
+    },
+    {
+      "description": "disable updates to the keycloak admin client - see https://github.com/janus-idp/backstage-plugins/issues/47 https://github.com/janus-idp/backstage-plugins/issues/1046", 
+      "matchPackageName": ["@keycloak/keycloak-admin-client"],
+      "groupName": "Keycloak dependency",
+      "enabled": "false"
+    },
+    {
+      "description": "ignore updates to the axios to version that keycloak 18 needs", 
+      "matchCurrentVersion": "0.26.1",
+      "matchPackageName": ["axios"],
+      "enabled": "false",
+      "groupName": "Keycloak dependency"
+    }    
   ],
   "ignorePaths": ["**/dist-dynamic/**"],
-  "ignoreDeps": ["@roadiehq/backstage-plugin-argo-cd"]
```

Comment:
why remove this? we want it pinned to 2.4.1 until we remove it in RHDH 1.4.

Reply:
It's not a dependency in this repo